### ISSUE TITLE
Add MFA to prior builds

### ISSUE THREAD

**Issue by brynwhyman** [silverstripe/cwp-recipe-kitchen-sink#38]
Given we're building the MFA modules to run on all SS4/CWP2 versions, we should add the modules to prior kitchen sink builds.

I would say it should be added to a CWP 1.9 build too, but that's not something we have at the moment, see #31

**Comment by robbieaverill** [comment]
The only reason we didn't do this before is that it requires PHP 7.1, so we'd need to add something custom into the Travis configuration (instead of composer dependencies) to install it

**Comment by robbieaverill** [comment]
I've added it to the CWP 2.2 builds and newer. I think it's reasonable that we suggest people use more recent recipe versions if they're adding MFA to a project. Even though CWP 2.0 is supported until 26/01/2020, SilverStripe 4.1 (which it uses) is already out of support, so presumably the support is limited to security and critical bugfixes only from CWP's perspective. CWP 2.2 uses SilverStripe 4.3 which is still in active core support.

>I would say it should be added to a CWP 1.9 build too, but that's not something we have at the moment, see #31

I've added that to the notes in #31
